test.sh

```diff
@@ -110,7 +110,7 @@ vagrantDestroy
 nodesOff
 
 # Power on vagrant box and nodes 
-vagrantUp
+vagrantUp "RackHD/packer/ansible/roles/monorail/files/auth_config"
```

Comment:
see the config.json in master https://github.com/RackHD/RackHD/blob/master/packer/ansible/roles/monorail/files/config.json